x/farming/client/cli/flags.go

```diff
@@ -28,15 +28,6 @@ func flagSetPlans() *flag.FlagSet {
 	return fs
 }
 
-// flagSetStakings returns the FlagSet used for farmer's staking coin denom.
-func flagSetStakings() *flag.FlagSet {
-	fs := flag.NewFlagSet("", flag.ContinueOnError)
-
-	fs.String(FlagStakingCoinDenom, "", "The staking coin denom")
-
-	return fs
-}
```

Comment:
Is there a reason why you erased it and hardcoded it?

Reply:
It's because there were too many `flagSet...()` functions that have same content(adding `FlagStakingCoinDenom` flag).
Since a flag set is to group flags by similar purpose, I decided to add flags directly to commands.